.vscode/settings.json

```diff
@@ -3,5 +3,10 @@
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "[svelte]": {
     "editor.defaultFormatter": "svelte.svelte-vscode"
+  },
+  "eslint.format.enable": true,
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true,
+    "source.organizeImports": true
```

Comment:
Ah great, this will enforce the same code styling in both of our VSCodes?

Reply:
Yes, I think it at least synchronizes some settings for code formatting plugins and import organization automatically for everyone. However I think you can still have different settings within these plugins.